### PR TITLE
[Standard] Test file indentation

### PR DESCRIPTION
**Pull request by pleportz**

### CODE REVIEW

File: code-quality/test-files-indentation.s.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Standardizing the indentation of a test file saves the developer 30 seconds per new function to test.
+
+## Checks
+
+The test file must have the following indentation even if only one function is tested:
+
+``` javascript
+describe('Name of the tested file', () => {
+  describe('function1', () => {
+    it('should do something', () => {
```
Comment (louiszawadzki) [inline code-quality/test-files-indentation.s.md]: `it` descriptions must not contain 'should'.
You wouldn't say out loud "I'm testing that Name of the tested file should do something", but "I'm testing that Name of the tested file does something"

So here your test name should be 'does something'


See Jest's official doc for example: https://facebook.github.io/jest/docs/en/api.html#testname-fn-timeout

File: code-quality/test-files-indentation.s.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Owner: [Pierre-Louis Le Portz](https://github.com/pleportz)
+
+## Why
+
+Standardizing the indentation of a test file saves the developer 30 seconds per new function to test.
+
+## Checks
+
+The test file must have the following indentation even if only one function is tested:
```
Comment (louiszawadzki) [inline code-quality/test-files-indentation.s.md]: You're giving one example but no clear rule of what it should be.

Can you describe the indentation in a few sentences that can be checked, for example:

- 1 `describe` per file
- 1 `describe` per function
- 1 `it` for each case

**Review by louiszawadzki — CHANGES_REQUESTED** [review]
See comments

File: code-quality/test-files-indentation.s.md (hunk 1 above)
Comment (pleportz) [reply code-quality/test-files-indentation.s.md]: 👍

**Comment by pleportz** [comment]
@louiszawadzki Merci beaucoup pour ta review :) 
J'ai fait des changements. Tu en penses quoi ?

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/indent-test-files-correctly/code-quality/test-files-indentation.s.md">test-files-indentation.s.md</a>**: You failed to mention a *Bad Examples*
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by louiszawadzki — APPROVED** [review]

**Comment by louiszawadzki** [comment]
@pleportz

**Pull request merged**